crates/node/src/service/rpc/metrics.rs

```diff
@@ -113,15 +113,15 @@ impl RpcMetrics {
         tracing::trace!(target: "rpc_metrics", "[{transport_label}] on_response started_at={:?}", now);
         tracing::trace!(target: "rpc_metrics::extra", "[{transport_label}] result={}", rp.as_result());
 
-        let micros = now.elapsed().as_micros();
+        let millis = now.elapsed().as_millis();
```

Comment:
my comment was about doing
```rs
        let time = now.elapsed();
        tracing::debug!(
            target: "rpc_metrics",
            "[{transport_label}] {} call took {:?}",
            req.method_name(),
            time,
        );

        self.calls_time.record(time.to_millis() as f64, &[KeyValue::new("method", req.method_name().to_string())]);
```
instead, using the Debug impl of Duration, not the Debug impl of u64

Reply:
Interesting, but don't you think this increases possibility of the two readings going out of sync, 
let's say due to xyz reasons a RPC call took `1 sec`, the `tracing` might record it as `1 sec` but the `calls_time` metric will record it as `1000 milliseconds`.
Whereas in the previous implementation both would have recorded `1000 milliseconds` 

WDYT ?